hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -331,7 +331,7 @@ public synchronized void close() throws IOException {
       }
     }
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Closing AbfsOutputStream ", toString());
+      LOG.debug("Closing AbfsOutputStream : {}", toString());
```

Comment:
switch to `LOG.debug("Closing AbfsOutputStream : {}", this)` and you can then remove the is debug enabled check.

Reply:
Done.